staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/validation/validation.go

```diff
@@ -543,6 +546,15 @@ func validateConversionReviewVersions(versions []string, requireRecognizedVersio
 	return allErrs
 }
 
+// Allows invalid CA Bundle to be specified only if the existing CABundle is invalid
+func allowInvalidCABundle(oldConversion *apiextensions.CustomResourceConversion) bool {
+	if oldConversion == nil || oldConversion.WebhookClientConfig == nil ||
+		len(oldConversion.WebhookClientConfig.CABundle) == 0 {
+		return false
+	}
+	return !webhook.IsValidCABundle(oldConversion.WebhookClientConfig.CABundle)
```

Comment:
I don't think we need two exported methods, this can call ValidateCABundle and check for non-zero error length

Reply:
Fixed